### PR TITLE
UI: make wifi network sorting more stable

### LINKED ISSUE #29375: UI: make wifi network sorting more stable
Networks are constantly moving around without any visible (in the four bars in the UI) signal strength change. If there's no difference in the four bars, the network shouldn't move.

### PR DESCRIPTION
**Pull request by deanlee**
resolve https://github.com/commaai/openpilot/issues/29375

### CODE REVIEW

**Review by adeebshihadeh — CHANGES_REQUESTED** [review]

File: selfdrive/ui/qt/offroad/wifiManager.h [hunk 1]
```diff
@@ -33,6 +33,7 @@ struct Network {
   SecurityType security_type;
 };
 bool compare_by_strength(const Network &a, const Network &b);
+inline int stretchLevel(unsigned int strength) { return std::clamp((int)round(strength / 33.), 0, 3); }
```
[the remark this answers is not in the document]
Comment (deanlee) [reply selfdrive/ui/qt/offroad/wifiManager.h:36]: the build will fail with "multiple definition error" if no inline here.

[the remark this answers is not in the document]
Comment (adeebshihadeh) [reply selfdrive/ui/qt/offroad/wifiManager.h:36]: why?

[the remark this answers is not in the document]
Comment (deanlee) [reply selfdrive/ui/qt/offroad/wifiManager.h:36]: because it's not a class member function and is included by multiple .cc files.

**Comment by adeebshihadeh** [comment]
Btw, seems like there's a bug where the currently connected network isn't highlighted in bold sometimes. Don't think it's a new thing though.

**Comment by deanlee** [comment]
Think it's because the font weight we used for the connected network was 500(QFont::Medium), which makes it look little different from the QFont::Normal(400):

>     #ssidLabel[disconnected=false] {
>       font-weight: 500;
>     }

Bolding the connected network in https://github.com/commaai/openpilot/pull/29385

**Pull request merged**